skyportal/handlers/api/lightcurve_fit.py

```diff
@@ -0,0 +1,166 @@
+import requests
+from baselayer.app.access import auth_or_token
+from baselayer.app.env import load_env
+
+from ..base import BaseHandler
+
+from ...models import (
+    DBSession,
+    LightcurveFit,
+    Obj,
+    Photometry,
+)
+
+env, cfg = load_env()
+
+if cfg['app.lc_fit.port'] is None:
+    LCFIT_URL = f"{cfg['app.lc_fit.protocol']}://{cfg['app.lc_fit.host']}"
+else:
+    LCFIT_URL = f"{cfg['app.lc_fit.protocol']}://{cfg['app.lc_fit.host']}:{cfg['app.lc_fit.port']}"
+
+
+class LightcurveFitHandler(BaseHandler):
+    """
+    ---
+    description: |
+        get the color and absolute magnitude of a source
+        based on cross-matches to some catalog (default is GAIA).
+    parameters:
+    - in: path
+        name: obj_id
+        required: true
+        schema:
+          type: string
+        description: ID of the object to retrieve photometry for
+      - in: query
+        name: model_name
+        required: false
+        schema:
+          type: string
+        description: |
+          The name of the model type to be fit.
+      - in: query
+        name: gptype
+        required: false
+        schema:
+          type: string
+        description: |
+          The type of interpolator to use for the model (if allowed).
+
+    responses:
+      200:
+        content:
+          application/json:
+            schema:
+              allOf:
+                  - $ref: '#/components/schemas/Success'
+                  - type: array
+                    items:
+                      type: object
+                      properties:
+                          origin:
+                            type: string
+                          color:
+                            type: float
+                          abs_mag:
+                            type: float
+
+      400:
+        content:
+        application/json:
+          schema: Error
+
+    """
+
+    @auth_or_token
+    def post(self, obj_id):
+        obj = Obj.query.get(obj_id)
+        if obj is None:
+            return self.error('Invalid object id.')
+
+        model_name = self.get_query_argument('model_name', 'Bu2019lm')
+        gptype = self.get_query_argument('gptype', 'tensorflow')
+
+        photometry = (
+            Photometry.query_records_accessible_by(self.current_user)
+            .filter(Photometry.obj_id == obj_id)
+            .all()
+        )
+        phot_data = []
+        for phot in photometry:
+            if phot.mag is not None:
+                phot_data.append(
+                    [
+                        phot.iso.format('YYYY-MM-DDTHH:mm:ss'),
+                        phot.filter.replace("ztf", ""),
+                        str(phot.mag),
+                        str(phot.e_mag),
+                    ]
```

Comment:
For the moment, I was just sending detections, to avoid the trail of limits before and after. My intuition is that ideally there would be somehow the ability to highlight the detections / limits you want to include and send those.